### PR TITLE
sed command update in spades_header_fixer/ tool

### PR DESCRIPTION
**Pull request by ConnorChato**
A Minor update to the sed command within spades_header_fixer.xml

I've attatched the test log here.
[spades_header_fixer_log.log](https://github.com/phac-nml/galaxy_tools/files/9365767/spades_header_fixer_log.log)

Tested with planemo on my home machine using the following steps

```
git clone https://github.com/phac-nml/galaxy_tools.git
cd galaxy_tools/tools/spades_header_fixer

virtualenv .venv; . .venv/bin/activate
pip install "pip>=7" # Upgrade pip if needed.
pip install planemo

planemo test --galaxy_root=../galaxy spades_header_fixer.xml
```

### CODE REVIEW

File: tools/spades_header_fixer/spades_header_fixer.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <requirements>
       <requirement type="package" version="4.7.0">sed</requirement>
   </requirements>
-	<command >sed -r "s/>NODE(_[0-9]+)_(.*)/>${input.element_identifier}\1 \2/g" $input >$output</command>
+  <command >sed -r "s/>NODE(_[0-9]+)_(.*)|>contig/>${input.element_identifier}\1 \2/g" $input >$output</command>
```
Comment (DarianHole) [inline tools/spades_header_fixer/spades_header_fixer.xml:6]: Thanks for the fix! Are you able to add in a second test in the `tests` block to check that this behaviour is working as intended?

**Review by DarianHole — CHANGES_REQUESTED** [review]
Thanks for the fix Connor! 

Just the one comment about tests.

Also, should the galaxy submodule be added in?

**Comment by apetkau** [comment]
I hope you do not mind me making a comment here. Since this changes the behaviour of the tool, I think we should also update the version number of the tool to `version="1.1.2"`.

In this case we can just bump the version number directly since the version isn't meant to match a version of a dependency (version of `sed`). But for cases where this is true (like SPAdes directly) you would update the version by adding a `+galaxy1` component. See https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

**Comment by DarianHole** [comment]
Oh thanks Aaron, I missed that

**Comment by ConnorChato** [comment]
Thanks for review - I just added the second test and updated the ver.

**Review by DarianHole — APPROVED** [review]
Thanks @ConnorChato!

**Pull request merged**